PixelHunter1995/Player.cs

```diff
@@ -62,20 +82,20 @@ public void Update(GameTime gameTime, InputManager input, bool controllable)
 
         public void HandleInput(InputManager input)
         {
-            if (input.GetState(InputCommand.PLAYING_Move).IsDown)
-            {
-                // Compensate for Position being in top left corner
-                float x = input.MouseSceneX - AnimationTileset.tileWidth / 2;
-                float y = input.MouseSceneY - AnimationTileset.tileHeight;
-                this.MovePosition = new Vector2(x, y);
-            }
             if (input.Input.GetKeyState(MouseKeys.LeftButton).IsEdgeDown)
```

Comment:
I guess you could still use PLAYING_Move. Or actually I guess it should be renamed to EXPLORING_Move.